src/auth.py

```diff
@@ -39,6 +40,8 @@ def _get_acls_from_cluster(self) -> str:
             f"--authorizer-properties zookeeper.connect={self.zookeeper}",
             "--list",
         ]
+        if self.ssl:
```

Comment:
So for the earlier comment, this could be `self.charm.tls.enabled` or whatever flag is appropriate.